detox/src/ios/earlgreyapi/GREYActions.js

```diff
@@ -196,60 +148,6 @@ starting from the given start points.
     };
   }
 
-  /*@return A GREYAction that scrolls to the given content @c edge of a scroll view.
-*/static actionForScrollToContentEdge(edge) {
```

Comment:
I would like to sanitize these enums. As the sanitizer isn't build yet I excluded them. My next goal is to readd these functions